build.rs

```diff
@@ -0,0 +1,59 @@
+use std::env;
+use std::fs;
+use std::process::Command;
+
+fn main() {
+  let mut curr_dir = env::current_dir().expect("Get project directory");
+  curr_dir.push("src");
+  curr_dir.push("parquet_thrift");
+
+  // Check if thrift is installed
+  if let Err(error) = run(Command::new("which").arg("thrift")) {
```

Comment:
We also need to check Thrift version here. Maybe we can do `thrift version` and check the major/minor version number.